### PR TITLE
Functional tests - Add test pagination order message

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add test pagination order message
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH='functional/BO/05_customerService/02_orderMessages/03_pagination' URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19074)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

File: tests/puppeteer/campaigns/functional/BO/05_customerService/02_orderMessages/03_pagination.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const OrderMessagesPage = require('@pages/BO/customerService/orderMessages');
+const AddOrderMessagePage = require('@pages/BO/customerService/orderMessages/add');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_customerService_order_messages_pagination';
```
Comment (Progi1984) [inline tests/puppeteer/campaigns/functional/BO/05_customerService/02_orderMessages/03_pagination.js]:
```suggestion
const baseContext = 'functional_BO_customerService_orderMessages_pagination';
```

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — COMMENTED** [review]
Need a smal change

File: tests/puppeteer/campaigns/functional/BO/05_customerService/02_orderMessages/03_pagination.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    await expect(pageTitle).to.contains(this.pageObjects.orderMessagesPage.pageTitle);
+  });
+
+  it('should reset all filters and get number of order messages in BO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+    numberOfOrderMessages = await this.pageObjects.orderMessagesPage.resetAndGetNumberOfLines();
+    if (numberOfOrderMessages !== 0) await expect(numberOfOrderMessages).to.be.above(0);
+  });
+
+  /* eslint-disable no-loop-func */
```
Comment (NeOMakinG) [inline tests/puppeteer/campaigns/functional/BO/05_customerService/02_orderMessages/03_pagination.js]: I seen that you disable it really often, maybe you could disable it in your eslint config if this is boring for you ? That would make sense 

ESLint configuration should be adapted to your need, if you consider that using loops is usefull for you.

Comment (Progi1984) [reply tests/puppeteer/campaigns/functional/BO/05_customerService/02_orderMessages/03_pagination.js]: Or use a correct code : https://eslint.org/docs/rules/no-loop-func (see samples)

Comment (boubkerbribri) [reply tests/puppeteer/campaigns/functional/BO/05_customerService/02_orderMessages/03_pagination.js]: To much vars out of scope (and linked to mocha too) to use the samples so I used a foreach

**Review by NeOMakinG — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**